source/Esf.Website/App/shared/layout/horizontalSplitView/horizontalSplitView.component.ts

```diff
@@ -0,0 +1,68 @@
+﻿import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+    selector: 'horizontal-split-view',
+    styleUrls: ['/App/shared/layout/horizontalSplitView/horizontalSplitView.styles.css'],
+    templateUrl: '/App/shared/layout/horizontalSplitView/horizontalSplitView.component.html',
+})
+export class HorizontalSplitViewComponent {
+    @Output() onResize: EventEmitter<any> = new EventEmitter();
+
+    private view1Height: string;
+    private view2Height: string;
+
+    private isDragging: boolean;
+
+    private containerNode: Node;
+
+    constructor() {
+        this.view1Height = "50%";
+        this.view2Height = "50%"
+        this.isDragging = false;
+    }
+
+    private onMouseDown(event: MouseEvent): void {
+        this.startDragging(event);
+    }
+
+    private onMouseMove(event: MouseEvent): void {
+        if (this.isDragging) {
+            this.recalculateHeigths(event);
+        }
+    }
+
+    private onMouseUp(event: MouseEvent): void {
+        this.stopDragging(event);
+    }
+
+    private onMouseLeave(event: MouseEvent): void {
+        this.stopDragging(event);
+    }
+    
+    private startDragging(event: MouseEvent): void {
+        this.isDragging = true;
+        this.containerNode = event.srcElement.parentNode;
+        document.body.style.cursor = 'ns-resize';
+    }
+
+    private stopDragging(event: MouseEvent): void {
+        this.isDragging = false;
+        this.containerNode = null;
+        document.body.style.cursor = 'auto';
+    }
+
+    private recalculateHeigths(event: MouseEvent): void {
+        let parentOffsetTop: number = (<any>this.containerNode).offsetTop;
```

Comment:
we should probably limit the minimum heights as right now one of the panes can disappear and you cannot resize it back to normal.